dp-cluster-setup-utility.py

```diff
@@ -1568,24 +1585,24 @@ def security_type_supported(self):
 class CMPrerequisites(BasePrerequisites):
   def __init__(self, cm):
     self.cm = cm
-    self.knox_host = cm.cluster.knox_host()
 
   def satisfied(self):
-    if not self.stack_supported():
-      print('The stack version (%s) is not supported. Supported stacks are: CDH-5.17/CDH-6.3 or newer.' % self.cm.installed_stack())
-      return False
+    for  cluster in self.cm.clusters:
+      if not self.stack_supported(cluster):
+        print('The stack version (%s) is not supported for %s. Supported stacks are: CDH-5.17/CDH-6.3 or newer.' % (cluster.installed_stack(), cluster.cluster_name))
+        return False
     return True
 
-  def stack_supported(self):
-    stack = self.cm.installed_stack()
+  def stack_supported(self, cluster):
+    stack = cluster.installed_stack()
     check_version = False
     (major,minor) = stack.version.split('.')[:2]
-    if (major == '5' and int(minor) >= 17) or (major == '6' and int(minor) >= 1):
+    if (major == '5' and int(minor) >= 17) or (major == '6' and int(minor) >= 3):
       check_version = True
     return stack.name == 'CDH' and check_version
 
   def security_type_supported(self):
```

Comment:
Can remove this for now.

Reply:
Should we remove both major and minor version check or only minor version?